Configure

```diff
@@ -12,10 +12,12 @@
 use 5.10.0;
 use strict;
 use Config;
+use FindBin;
+use lib "$FindBin::Bin/util/perl";
```

Comment:
I suppose this flies on VMS... I mean slashes, and you don't have to build path with some platform-neutral call...